daemon/logger/factory.go

```diff
@@ -119,6 +119,7 @@ func GetLogDriver(name string) (Creator, error) {
 var builtInLogOpts = map[string]bool{
 	"mode":            true,
 	"max-buffer-size": true,
+	"max-log-size":    true,
```

Comment:
Given that there has been `max-buffer-size`, I wonder `max-log-size` seems confusing .
How about `max-copier-buffer-size` or something else? cc @cpuguy83 @thaJeztah

Reply:
What about `max-log-length` or `max-line-length`? I'm afraid the concept of `copier` is an implementation detail that would be hard to surface in documentation. WDYT?

Reply:
`max-line-length` looks fine